packages/utilities/psammead-storybook-helpers/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 8.1.5 | [PR#2704](https://github.com/bbc/psammead/pull/2704) Add exported styled paragraph to be wrapped with un-styled components, i.e inline link. |
```

Comment:
```suggestion
| 9.0.0 | [PR#2704](https://github.com/bbc/psammead/pull/2704) Add exported styled paragraph to be wrapped with un-styled components, i.e inline link. |
```
I would consider this a breaking change since we're completely changing the exported value.